TaskMonitorPlus/__init__.py

```diff
@@ -99,6 +99,21 @@ def __stop__(self):
         eg.messageReceiver.RemoveHandler(WM_APP + 2, self.WindowCreatedProc)
         eg.messageReceiver.RemoveHandler(WM_APP + 3, self.WindowDestroyedProc)
 
+    def CheckPlugins(self):
+        for plugin in eg.pluginList:
+            if (
+                plugin.info.guid == "{D1748551-C605-4423-B392-FB77E6842437}"
+                and plugin.info.treeItem.isEnabled
```

Comment:
we also want to make sure that the treeItem is not already disabled for the plugin this way we will disable it if it is enabled.